services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/DiscoveryMechanismSteps.java

```diff
@@ -20,12 +20,12 @@
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 
-import org.openkilda.atdd.staging.model.topology.TopologyDefinition;
-import org.openkilda.atdd.staging.service.floodlight.FloodlightService;
-import org.openkilda.atdd.staging.service.floodlight.model.FlowEntriesMap;
-import org.openkilda.atdd.staging.service.floodlight.model.SwitchEntry;
-import org.openkilda.atdd.staging.steps.helpers.DefaultFlowsChecker;
-import org.openkilda.atdd.staging.steps.helpers.TopologyChecker.SwitchEntryMatcher;
+import org.openkilda.atdd.staging.helpers.DefaultFlowsChecker;
+import org.openkilda.atdd.staging.helpers.TopologyChecker.SwitchEntryMatcher;
+import org.openkilda.testlib.model.topology.TopologyDefinition;
```

Comment:
Package doesn't not always equal a module or library. What about naming the package as `testing`, then it can be shared with another modules or libraries?